src/bundle/Resources/views/themes/admin/content_type/field_definitions.html.twig

```diff
@@ -1,5 +1,26 @@
 <div class="ibexa-content-type-edit__field-definitions-group">
-    <div class="ibexa-content-type-edit__field-definitions-group-list">
+    {% set extra_actions = [
+        {
+            'icon_name': 'discard',
+            'icon_size_class': 'ibexa-icon--tiny-small',
+            'button_class': 'ibexa-collapse__extra-action-button--remove-field-definitions',
+        }
+    ] %}
+
+    <div
+        class="ibexa-content-type-edit__field-definitions-group-list"
+        data-template="{{ include("@ibexadesign/ui/component/collapse.html.twig", {
+            'is_expanded': false,
+            'is_draggable': true,
+            'class': 'ibexa-collapse--field-definition ibexa-collapse--field-definition-highlight ibexa-collapse--field-definition-loading',
+            'body_id': 'loading-collapse',
+            'header_label': 'content_type.view.edit.default_header'|trans|desc('New field type') ~ ' ({{ type }})',
```

Comment:
```suggestion
            'header_label': 'content_type.view.edit.default_header'|trans({ '%type%': type })|desc('New field type (%type%)'),
```

Reply:
current version is ok - it's generic template and inside JS `{{ type }}` is replaced dynamically